packages/@aws-cdk/aws-eks/README.md

```diff
@@ -1,31 +1,52 @@
 ## Amazon EKS Construct Library
-
 <!--BEGIN STABILITY BANNER-->
 ---
 
 ![cfn-resources: Stable](https://img.shields.io/badge/cfn--resources-stable-success.svg?style=for-the-badge)
 
 > All classes with the `Cfn` prefix in this module ([CFN Resources](https://docs.aws.amazon.com/cdk/latest/guide/constructs.html#constructs_lib)) are always stable and safe to use.
 
-![cdk-constructs: Experimental](https://img.shields.io/badge/cdk--constructs-experimental-important.svg?style=for-the-badge)
+![cdk-constructs: Developer Preview](https://img.shields.io/badge/cdk--constructs-developer--preview-informational.svg?style=for-the-badge)
 
-> The APIs of higher level constructs in this module are experimental and under active development. They are subject to non-backward compatible changes or removal in any future version. These are not subject to the [Semantic Versioning](https://semver.org/) model and breaking changes will be announced in the release notes. This means that while you may use them, you may need to update your source code when upgrading to a newer version of this package.
+> The APIs of higher level constructs in this module are in **developer preview** before they become stable. We will only make breaking changes to address unforeseen API issues. Therefore, these APIs are not subject to [Semantic Versioning](https://semver.org/), and breaking changes will be announced in release notes. This means that while you may use them, you may need to update your source code when upgrading to a newer version of this package.
 
 ---
 <!--END STABILITY BANNER-->
 
-This construct library allows you to define [Amazon Elastic Container Service
-for Kubernetes (EKS)](https://aws.amazon.com/eks/) clusters programmatically.
-This library also supports programmatically defining Kubernetes resource
-manifests within EKS clusters.
+This construct library allows you to define [Amazon Elastic Container Service for Kubernetes (EKS)](https://aws.amazon.com/eks/) clusters.
+In addition, the library also supports defining Kubernetes resource manifests within EKS clusters.
+
+* [Quick Start](#quick-start)
+* [API Reference](https://docs.aws.amazon.com/cdk/api/latest/docs/aws-eks-readme.html)
+* [Architectural Overview](#architectural-overview)
+* [Provisioning clusters](#provisioning-clusters)
+    * [Capacity Types](#capacity-types)
+      * [Managed Node Groups](#1-managed-node-groups)
+      * [Fargate Profiles](#2-fargate-profiles)
+      * [Self Managed Auto Scaling Groups](#3-self-managed-auto-scaling-groups)
+    * [ARM64 Support](#arm64-support)
+    * [Kubectl Support](#kubectl-support)
```

Comment:
Why is this in "provisioning clusters"

Reply:
Because it describes the `kubectlLayer` and `kubectlEnvironment` properties which are used to configure new clusters.

Reply:
Feels like this should be under the "applying manifests" given this is where it's relevant

Reply:
It will be a bit awkward since its relevant for helm charts as well. I think its ok to introduce this here as part of the cluster configuration, we already mentioned that this library supports applying manifests, so its not completely out of context. I'll add a link to this from the applying manifests section as well.

k?